list_to_email_converter.py

```diff
@@ -0,0 +1,178 @@
+    for object in names:
+        if(object != ""):
+            pms.append(object)
+            people.append(object)
+    link_pair[project]=pms
+
+
+people = sorted(people)
+people.append("buffer") ##buffer element for loop
+num_lines = sum(1 for line in open('_data/people.yml'))
+
+
+email = open('_data/people.yml', 'r')
+
+akt_lines=0
+
+i=0
+while i < len(people)-1:
+    if people[i].find('*') != -1:
+        people[i] = people[i].replace("*" , "")
+        j=i+1
+        while people[j].find('*') != -1 and people[i] == people[j].replace("*", "") :
+            people[j] = people[j].replace("*", "")
+            j= j+1
+        ## if one person is projectleader of more than one project
+        while people[i]==people[i+1]:
+            people.pop(i+1)
+        while people[i] == people[i -1]:
+            people.pop(i-1)
+            i=i-1
+        ## if one projectleader is involved in other projects
+        helpvar = 0
+        name = ""
+        emailaddress = ""
+        for line in email:
+            akt_lines = akt_lines+1
+            if (line ==''):
+                email.close()
+                email = open('_data/people.yml', 'r')
+            if line.find(people[i]+":")!= -1:
+                helpvar = 1
+            elif helpvar > 0 and line.find('  sur_name:') != -1:
+                name = name + line.replace("  sur_name: ", "").rstrip()
+                helpvar = helpvar + 1
+            elif helpvar > 0 and line.find('  given_name:') != -1:
+                name = line.replace("  given_name: ", "").rstrip() + " " + name
+                helpvar = helpvar + 1
+            elif helpvar > 0 and line.find('  email:') != -1:
+                emailaddress = line.replace("  email: ", "").rstrip()
+                helpvar = helpvar + 1
+            elif helpvar == 4:
+                break
+            if (akt_lines >= num_lines):
+                email.seek(0)
+                akt_lines = 0
+
+        head.append([people[i], name, emailaddress])
+        realname[people[i]] = name
+        i=i+1
+        ## checks email and real name of the projecleaders
+
+    else:
+        while people[i] == people[i + 1]:
+            people.pop(i+1)
+        if people[i+1].find('*') != -1 and people[i] == people[i+1].replace("*", ""):
+            i=i+1
+            continue
+            ## if one person is projectleader
+        helpvar = 0
+        name = ""
+        for line in email:
+            akt_lines = akt_lines + 1
+            if (line == ''):
+                email.close()
+                email = open('_data/people.yml', 'r')
+            if line.find(people[i]) != -1:
+                helpvar = 1
+            elif helpvar > 0 and line.find('  sur_name:') != -1:
+                name = name + line.replace("  sur_name:", "").rstrip()
+                helpvar = helpvar + 1
+            elif helpvar > 0 and line.find('  given_name:') != -1:
+                name = line.replace("  given_name: ", "").rstrip() + " " + name
```

Comment:
There are now two blanks between the first and the last name.. one is enough. Is it added here?